### DISCUSSION TITLE
Status pooling but no stats

### DISCUSSION THREAD

**Discussion by evlaboz** [Chia-Network/chia-blockchain#7441] [Support]
I am having some issue wit GUI, i generated 10 plots for pool farming on pool.space but all stats are still 0 after 24h. Status is pooling but when i try to find farmer on https://mainnet.pool.space/ with lancher id, I got this Unable to find details for this farm in Space Pool.

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
